### PR TITLE
PLT-2971 Added email footer for address

### PR DESCRIPTION
**Pull request by DavidLu1997**
Previously, all our emails had Mattermost in the footer. That is now changed to be user configurable through the admin console.

Before: `Sent by (c) 2015 Mattermost, Inc. 855 El Camino Real, 13A-168, Palo Alto, CA, 94301`
Now: `Sent by 1337 I like cats` or whatever the user wants

### CODE REVIEW

File: model/config.go [hunk 1]
```diff
@@ -123,6 +123,7 @@ type EmailSettings struct {
 	RequireEmailVerification bool
 	FeedbackName             string
 	FeedbackEmail            string
+	FeedbackOrganization     *string
```
Comment (coreyhulen) [inline model/config.go]: Need default added to `config.SetDefaults()` when nil

**Comment by DavidLu1997** [comment]
Default added

File: model/config.go (hunk 1 above)
Comment (coreyhulen) [inline model/config.go]: Shouldn't the default be `(c) 2015 Mattermost, Inc. 855 El Camino Real, 13A-168, Palo Alto, CA, 94301.` ??  maybe ask a PM.

File: utils/html.go [hunk 2]
```diff
@@ -86,6 +86,14 @@ func (t *HTMLTemplate) addDefaultProps() {
 	}
 
 	t.Props["Footer"] = localT("api.templates.email_footer")
+	t.Props["Organization"] = localT("api.templates.email_organization")
```
Comment (crspeller) [inline utils/html.go]: This would be less confusing if you did the localT call down where your constructing the string, instead of appending.

**Comment by DavidLu1997** [comment]
Changes according to JB:
- Default set to empty, if it is empty, do not display a footer
- Updated examples

**Comment by enahum** [comment]
+1

**Comment by hmhealey** [comment]
+1 again

**Pull request merged**